libs/sdk-ui-filters/src/DateFilter/Select/Select.tsx

```diff
@@ -66,11 +69,3 @@ export const Select = <V extends {}>({
         </TypedDownshift>
     );
 };
-
-Select.defaultProps = {
```

Comment:
These are reported as red warning in browser console as soon to be deprecated by future version of react. It made my console unreadable so I got rid of them.